.github/workflows/release.yml

```diff
@@ -0,0 +1,103 @@
+name: ci
+
+# This worflow needs this secret:
+#
+# DOCKERPASSWORD = Docker Hub token
```

Comment:
I think I would prefer to name this `DOCKER_TOKEN`, like in the action's usage documentation.

Reply:
Sure, you can do this. I had it that way earlier. But I tried to reduce the amount of secrets to make it easier for me. And as at least my dockerhub username is part of the image name itself, I didn't see the need to hide it. But yes, this could be changed.